ntbs-service/Services/ReportingLinksService.cs

```diff
@@ -23,8 +23,15 @@ public ReportingLinksService(IConfiguration configuration)
 
         public string GetClusterReport(string clusterId)
         {
-            // TODO Implement this properly in NTBS-2249
-            return "";
+            var clusterReportBase = _externalLinks.ClusterReportBase;
+            var clusterReportReplacementSymbol = _externalLinks.ClusterReportReplacementSymbol;
+
+            if (string.IsNullOrEmpty(clusterReportBase) || string.IsNullOrEmpty(clusterReportReplacementSymbol))
+            {
+                return "";
+            }
+
+            return clusterReportBase.Replace(clusterReportReplacementSymbol, clusterId);
```

Comment:
Also, it might be a little more readable written as a ternary, but that is a matter of opinion.

Reply:
Yep, done!